### PR TITLE
systemd_cgroups_t: fix denials

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: policy/modules/system/systemd.te [hunk 1]
```diff
@@ -182,12 +182,17 @@ fs_register_binary_executable_type(systemd_binfmt_t)
...
+
+# write to /run/systemd/cgroups-agent
+allow systemd_cgroups_t init_t:unix_dgram_socket sendto;
+
 kernel_domtrans_to(systemd_cgroups_t, systemd_cgroups_exec_t)
 kernel_dgram_send(systemd_cgroups_t)
 
 init_stream_connect(systemd_cgroups_t)
 
-systemd_log_parse_environment(systemd_cgroups_t)
```
Comment (pebenito) [inline policy/modules/system/systemd.te]: This line was in the correct position.

File: policy/modules/system/systemd.te [hunk 2]
```diff
@@ -182,12 +182,17 @@ fs_register_binary_executable_type(systemd_binfmt_t)
...
+selinux_getattr_fs(systemd_cgroups_t)
```
Comment (pebenito) [inline policy/modules/system/systemd.te]: This line goes above `init_stream_connect()`

**Comment by cgzones** [comment]
should be properly ordered now
i move the `systemd_log_parse_environment` line in the first place, cause of the local nature of this interface

File: policy/modules/system/systemd.te [hunk 3]
```diff
@@ -182,9 +182,14 @@ fs_register_binary_executable_type(systemd_binfmt_t)
 # Cgroups local policy
 #
 
+# write to /run/systemd/cgroups-agent
+allow systemd_cgroups_t init_t:unix_dgram_socket sendto;
```
Comment (pebenito) [inline policy/modules/system/systemd.te]: `init_t` isn't in this module.

**Comment by cgzones** [comment]
fixed usage of init_t

**Pull request merged**